website/content/en/highlights/2022-10-04-0-25-0-upgrade-guide.md

```diff
@@ -40,6 +41,43 @@ After:
 mod(5, 2) == 1
 ```
 
+#### Removal of the `new_relic_logs` sink {#new-relic-logs-sink-removal}
```

Comment:
I think we should backport these instructions to 0.24.0 to enable more users to migrate during the deprecation period. We could then just reference them in this 0.25.0 upgrade guide.

Otherwise this guide looks good!

Reply:
Yeah, we could do that in the same PR to add the warning log to `new_relics_logs` in 0.24.1.

Reply:
👍 I like it.